### PR TITLE
Specify zoom-in shortcut

### PR DESCRIPTION
**Pull request by gsusmi**
[Bug](https://github.com/electron/electron/issues/15496) in electron forces zoom-in shortcut to be defined for keyboard shortcut to work.

### CODE REVIEW

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## Next
+
+### Changes
```
Comment (randallmorey) [inline ui/desktop/CHANGELOG.md:7]: Is this a change or more like a fix?

File: ui/desktop/tests/acceptance/origin-test.js [hunk 2]
```diff
@@ -178,12 +178,15 @@ module('Acceptance | origin', function (hooks) {
...
-    assert.expect(4);
-    assert.notOk(mockIPC.origin);
+    assert.expect(3);
     await visit(urls.origin);
+    await a11yAudit();
     await fillIn('[name="host"]', window.location.origin);
     await click('[type="submit"]');
-    assert.equal(mockIPC.origin, window.location.origin);
+    assert.equal(
+      this.owner.lookup('controller:origin').origin,
```
Comment (randallmorey) [inline ui/desktop/tests/acceptance/origin-test.js:187]: This seems unrelated to the PR.  I'm not seeing an `origin` controller in Desktop so I'm curious where this comes from and why the change makes sense.  Can this go in a separate PR?

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: ui/desktop/tests/acceptance/origin-test.js (hunk 2 above)
Comment (gsusmi) [reply ui/desktop/tests/acceptance/origin-test.js:187]: Sure. I'll break this out into a new PR.

File: ui/desktop/CHANGELOG.md (hunk 1 above)
Comment (gsusmi) [reply ui/desktop/CHANGELOG.md]: Moved to 'Bug Fixes' category.

**Review by randallmorey — APPROVED** [review]
